scripts/process.py

```diff
@@ -11,21 +17,17 @@
 DEATH_US = 'time_series_covid19_deaths_US.csv'
 REFERENCE = 'https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/UID_ISO_FIPS_LookUp_Table.csv'
 
+
+## Define functions to clean data
```

Comment:
This is self-explanatory. Besides, function comments should be located inside functions:

```
def my_function():
    '''My comment here.'''
```

Here, the code is simple, but generally that doesn't help in knowing to which part of the code the comment applies when left outside of code blocks. What if functions are added below and have a different purpose?